lib/browser/api/dialog.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
     if (options.buttonLabel == null) {
       options.buttonLabel = ''
     } else if (typeof options.buttonLabel !== 'string') {
-      throw new TypeError('buttonLabel must be a string')
+      throw new TypeError('Button label must be a string')
```

Comment:
Changing this to be consistent with the other errors in this file like `Default path`